projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -178,6 +168,28 @@ public static function permissions_callback() {
 		return current_user_can( 'manage_options' );
 	}
 
+	/**
+	 * Return true if we should initialize the My Jetpack
+	 */
+	public static function should_initialize() {
+		if ( did_action( 'my_jetpack_init' ) ) {
+			return false;
+		}
+		$should = apply_filters( 'jetpack_my_jetpck_should_initialize', true );
```

Comment:
typo

Reply:
lol

Reply:
fixed in 5abdf4960bc749ce99534b34045914297ec18960 thanks!